README[PTBR].md

```diff
@@ -0,0 +1,41 @@
+# Jogo da memória
```

Comment:
Este README ficou top top top! Se quiser, coloca instruções de como subir o projeto localmente para quem quiser testar. Ah! Coloca também o link para o seu projeto rodando se já estiver no ar.

Reply:
To pra fazer isso mesmo
Vou atualizar também a versão do GitPages, ele ta com uma versão mais antiga, só com o jogo. Aí vou adicionar a tela de login, signup e error com as instruções no README

Reply:
Ah esqueci de comentar, você colocou sem licença então quer dizer que seu código é de domínio público e aí você perde inclusive direito autoral sobre ele. Eu recomendo você colocar as licenças de open source pelo menos, no mínimo uma **Apache V2**.